### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader : Proper link text for controls such as "Visual studio ...", "installs ..." and "Azure pipelines ..." are not announced for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-azurefunctions#2286]
**“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

GitHub Tags:#A11y_Azure Extensions for VS Code;#Azure Tools for VS Code;#A11yTCS;#A11yMAS;#MAC;#FTP;#Benchmark;#Screenreader;#VoiceOver;#A11ySev1;#MAS1.3.1;#MAS4.1.2;#MAS1.1.1;

### Environment Details:
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.3.1, MAS4.1.2, MAS1.1.1

### Repro Steps:
1. Open Visual studio code
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button, the "Default Screen" will appear.
4. Start screen reader and navigate to controls such as controls such as "Visual studio ...", "installs ..." and "Azure pipelines ..." and observe if the visual text is announced or not.

### Actual:
When screen reader users navigate to the controls such as controls such as "Visual studio marketplace v0.23.0", "installs 550.21k" and "Azure Pipelines failed" there visual text is not announced properly to the screen reader users which is incorrect. Since these controls are image links the alternative text provided should be descriptive for screen reader users.

### Expected:
The controls such as "Visual studio marketplace v0.23.0", "installs 550.21k" and "Azure Pipelines failed" should be announced with the proper name as per its visual appearance on the screen for screen reader users when they navigate to these controls.

### User Impact:
If proper name is not announced for the interactive elements on the screen content then the screen reader users will not understand how to interact with these control and this can lead to the confusion for screen reader users.

### Attachment link for Reference:
[2286_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader Proper link text for controls such as "Visual studio ...", "installs ..." and "Azure pipelines ..." are not announced for screen reader users.zip](https://github.com/microsoft/vscode-azurefunctions/files/5042250/2286_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader.Proper.link.text.for.controls.such.as.Visual.studio.installs.and.Azure.pipelines.are.not.announced.for.screen.reader.users.zip)

**Comment by EricJizbaMSFT** [comment]
Those are not necessary, so I just removed them.

<img width="693" alt="Screen Shot 2020-08-25 at 3 27 51 PM" src="https://user-images.githubusercontent.com/11282622/91234167-8f0c4000-e6e7-11ea-9f50-d81eebc1112a.png">

**Comment by MayuriNaik** [comment]
Verified the bug in Visual Studio Code - Insiders application (Version: 1.50.0-insider) and Azure Functions (v0.24.0). The controls such as "Visual studio marketplace v0.23.0", "installs 550.21k" and "Azure Pipelines failed" are now no more available. Hence closing the bug. PFA
<img width="1116" alt="2286_Closed" src="https://user-images.githubusercontent.com/52036772/93176166-77f0c900-f74e-11ea-85ae-b8769609c0f1.png">
